src/common/components/Table/TableHeaderRow.tsx

```diff
@@ -0,0 +1,19 @@
+import classNames from 'classnames';
```

Comment:
lets do it this way import cn from 'classnames', because it's shorter and is not confusing as much as className classNames